Makefile.am

```diff
@@ -58,8 +57,11 @@ ChangeLog: $(srcdir)/tools/git2cl
 		$(srcdir)/tools/git2cl > $@ \
 	; fi
 
+indent:
+	cd src && $(MAKE) $(AM_MAKEFLAGS) indent
```

Comment:
No, this one isn't for that. This "cd src" is to run the "indent" target from the "src/Makefile", which then cd's to $(top_srcdir). You know this "indent" target is non-recursive, right? So if we run "make indent" in top_srcdir, let it descend to "src" directory to do the job.